### DISCUSSION TITLE
I am having some problems with a code

### DISCUSSION THREAD

**Discussion by antoniojoao1** [microsoft/Power-Fx#2185] [Q&A]
Hey there guys, I am new with power fx and I am having a problem, so there is this checklist, I need to make some adjustments. It's a vehicle checklist, and I need to add a function that prevents users to open a new checklist without closing a previous checklist, so I developed this code: 
If(
    IsBlank(
        LookUp(
            Responsavel_DataCard2,
            DataCardKey5 = DataCardValue30 And ChecklistStatus = "Open"
        )
    ),
    Notify("A new checklist can be open."),
    Notify("You have an open checklist. Finalize it before opening a new one.")
)

but power apps keep saying there is an error on the code, and I can't find, any help would be welcomed.

**Comment by anderson-joyle** [comment]
Can you please share what the error message says?
Also, I suggest you ask this question at https://powerusers.microsoft.com/t5/Building-Power-Apps/bd-p/PowerAppsForum1.

**Reply by antoniojoao1** [reply]
Hey, Anderson. When I apply the code, I get a lot of errors, even in itens there are directed affected by the code. I get "Unexpected 
Characters", I get "Invalid Name". The "unexpected characters" erros says that I should change an at sign to a colon. The "invalid name" says that some item does not exist anymore, but when I remove the code, all of the errors disappears.
